lib/internal/child_process.js

```diff
@@ -498,9 +498,9 @@ function setupChannel(target, channel) {
       var queue = target._handleQueue;
       target._handleQueue = null;
 
-      queue.forEach(function(args) {
+      for (var args in queue) {
```

Comment:
Does this work? `for (var … in …)` iterates over the indices of the array, so it does not do the same thing as `for (var i = 0; i < args.length; ++i)`